site/config.toml

```diff
@@ -0,0 +1,166 @@
+    url = "/docs/"
+  [[menu.main]]
+    name = "GitHub"
+    weight = -99
+    pre = "<i class='fab fa-github pr-2'></i>"
+    url = "https://github.com/kubernetes-sigs/kueue"
+
+###############################################################################
+# Docsy - Goldmark markdown parser
+###############################################################################
+[markup]
+  [markup.goldmark]
+    [markup.goldmark.renderer]
+      unsafe = true
+  [markup.highlight]
+    # See a complete list of available styles at https://xyproto.github.io/splash/docs/all.html
+    style = "tango"
+    guessSyntax = "true"
+
+###############################################################################
+# Docsy - Image processing configuration
+###############################################################################
+[imaging]
+  resampleFilter = "CatmullRom"
+  quality = 75
+  anchor = "smart"
+
+###############################################################################
+# Docsy - Services configuration
+###############################################################################
+[services]
+  [services.googleAnalytics]
+  id = "UA-135379910-1"
+
+###############################################################################
+# Docsy - Language configuration
+###############################################################################
+[languages]
+  [languages.en]
+    title = "Kueue"
+    languageName ="English"
+    # weight used for sorting in the language menu
+    weight = 1
+
+###############################################################################
+# Docsy - Site Parameters
+###############################################################################
+[params]
+  github_repo = "https://github.com/kubernetes-sigs/kueue"
+  github_project_repo = "https://github.com/kubernetes-sigs/kueue"
+  github_subdir = "site"
+  RSSLink = "/index.xml"
+  author = "kubernetes" # add your company name
+  github = "kubernetes" # add your github profile name
+  twitter = "kubernetesio" # add your twitter profile
+
+  copyright = "The Kubernetes Authors"
+  privacy_policy = "https://policies.google.com/privacy"
+
+  # Google Custom Search Engine ID.
+  # gcs_engine_id = "007239566369470735695:624rglujm-w"
+  prism_syntax_highlighting = true
+  # Text label for the version menu in the top bar of the website.
+  version_menu = "Version"
+
+  # The major.minor version tag for the version of the docs represented in this
+  # branch of the repository. Used in the "version-banner" partial to display a
+  # version number for this doc set.
+  version = "main"
+
+  # Flag used in the "version-banner" partial to decide whether to display a
+  # banner on every page indicating that this is an archived version of the docs.
+  archived_version = false
+
+  # A link to latest version of the docs. Used in the "version-banner" partial to
+  # point people to the main doc site.
+  url_latest_version = "https://sigs.k8s.io/kueue/docs"
```

Comment:
We are settled on removing the existing docs.

Update this to https://sigs.k8s.io/kueue/site/content/en/docs until we have a domain.